<div class="chatlog__message-group">
  <div id="chatlog__message-container-542763164106883073" class="chatlog__message-container" data-message-id="542763164106883073">
    <div class="chatlog__message">
      <div class="chatlog__message-aside"><img class="chatlog__avatar" src="https://cdn.discordapp.com/avatars/296587674737704960/5d3c584499ce245b7129cda8c7ae00e9.png?size=512" alt="Avatar" loading="lazy"></div>
      <div class="chatlog__message-primary">
        <div class="chatlog__header"><span class="chatlog__author" title="_rintaro" data-user-id="296587674737704960">rintaro</span> <a href="/channels/430242233468452865?category=main&amp;channel=swiftbot-sandbox&amp;message_id=542763599714582530"><span class="chatlog__timestamp" title="Wednesday, 06 February 2019 17:47"></span></a><a href="#chatlog__message-container-542763164106883073">02/06/2019 17:47</a></div>
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve">
            <ul>
              <li>single expression closure ではクロージャの返値が Void の時でも、非Void のexpressionを書ける</li>
              <li><code class="chatlog__markdown-pre chatlog__markdown-pre--inline">{ ... in return &lt;expr&gt; }</code> は single expression closure と解釈される</li>
            </ul>という事実が重なって起きてる不幸ですね
          </span> <span class="chatlog__edited-timestamp" title="Wednesday, 06 February 2019 23:22">(edited)</span></div>
      </div>
    </div>
  </div>
  <div id="chatlog__message-container-542763599714582530" class="chatlog__message-container" data-message-id="542763599714582530">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <div class="chatlog__short-timestamp" title="Wednesday, 06 February 2019 17:48">17:48</div>
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve"><code class="chatlog__markdown-pre chatlog__markdown-pre--multiline language-swift">let _ : () -&gt; Void = { return "test" }</code> が通るの自体はもはや仕様なので、エラー診断のバグという扱いになると思います。</span></div>
      </div>
    </div>
  </div>
</div>